app/src/pages/inside/logsPage/logItemInfo/InfoTabs/logItemInfoTabs.scss

```diff
@@ -0,0 +1,113 @@
+.tabs-container {
+}
+
+.tabs {
+  display: flex;
+  background-color: $COLOR--white;
+  border-top: 1px solid $COLOR--gray-91;
+
+  @media (max-width: $SCREEN_XS_MAX) {
+    flex-direction: column;
+  }
+}
+
+.tab {
+  display: flex;
+  align-items: center;
+  position: relative;
+  padding: 0 10px;
+  cursor: pointer;
+  color: $COLOR--gray-47;
+  fill: $COLOR--gray-47;
+  transition: fill 0.2s, color 0.2s;
+  height: 40px;
+  box-sizing: border-box;
+  background-color: transparent;
+  border: 0;
+  outline: 0;
+
+  &:hover {
+    color: #000;
+    fill: #000;
```

Comment:
On our project all colors should be defined with color variables from color palette. If you see the color which is not represented in color palette - discuss it with our UX Designer.

Reply:
Discussing with designers...

Reply:
We decided that we should use $COLOR--charcoal-grey here.